### PR TITLE
Feature/create mail sender with rabbitmq

### PR DESCRIPTION
**Pull request by EugeneTryF**
created mail sender + rebbitMq + changes in integration tests cause of rabbit

### CODE REVIEW

File: src/Notofications/Notifications.API/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "RabbitMq": {
+    "RabbitMqHost": "localhost",
+    "RabbitMqQueue": "MyLocalQueue"
+  },
+  //TODO: dont forgot to change this settings before sending emails
```
Comment (VadimZharikov) [inline src/Notofications/Notifications.API/appsettings.json:6]: ?

**Review by VadimZharikov — CHANGES_REQUESTED** [review]

File: src/Notofications/Notifications.API/appsettings.json (hunk 1 above)
Comment (EugeneTryF) [reply src/Notofications/Notifications.API/appsettings.json:6]: dont know where to store password and dont want to save it on pc os in keys

Comment (EugeneTryF) [reply src/Notofications/Notifications.API/appsettings.json:6]: thats why change it myself if need notification service

**Review by Genresal — APPROVED** [review]
LGTM!

**Review by v12code — APPROVED** [review]

**Pull request merged**